figures/permissions.py

```diff
@@ -3,11 +3,91 @@
 '''
 from rest_framework.permissions import BasePermission
 
+import django.contrib.sites.shortcuts
 
-class IsStaffUser(BasePermission):
+import organizations
+
+import figures.settings
+
+
+class MultipleOrgsPerUserNotSupported(Exception):
```

Comment:
Someone will get this error, sooner or later. It helps to add some docs here to specify why this is not supported, and if there's support.

Reply:
nvm, found the why :point_down: